lua/refactoring/tests/refactor/106/lua/assign-func/extract.start.lua

```diff
@@ -0,0 +1,11 @@
+-- stylua: ignore start
```

Comment:
Technically we don't really need this anymore because I added a .styluaignore a bit ago, but it's fine - I'm going to make another PR soon anyway to get rid of these at the top of all the lua tests.

Reply:
Oh didn't know you added that, nice!